ts/screens/profile/ServicesPreferenceScreen.tsx

```diff
@@ -81,7 +116,7 @@ const ServicesPreferenceScreen = (props: Props): React.ReactElement => {
         <ContentWrapper>
```

Comment:
Are we missing the `SafeAreaView` here?

Reply:
in this case no because the RNavScreenWithLargeHeader component already uses a spacing provided by [useSafeAreaInsets exported by react-native-safe-area-context](https://github.com/pagopa/io-app/blob/9c4c4d14cedbe79a2d3738ffc9ce0ad73e6cb402/ts/components/ui/RNavScreenWithLargeHeader.tsx#L67)